oktoberfest/plotting/plotting.py

```diff
@@ -289,6 +289,9 @@ def plot_sa_distribution(prosit_df: pd.DataFrame, target_df: pd.DataFrame, decoy
     :param filename: the path to the location used for storing the plot
     """
     _, _, _, psm_col = _check_columns(target_df)
+
```

Comment:
why? This has nothing to do with anndata right? If you encountered a breaking change after using anndata here, it is because anndata may store the metadata differently in the .obs df? Then you would need to check it is correctly stored to solve potentially upstream issues of this issue